.github/workflows/project-automation.yml

```
@@ -0,0 +1,78 @@
+                    input: {
+                      projectId: $projectId
+                      itemId: $itemId
+                      fieldId: "Status"
+                      value: { singleSelectOptionId: "${newStatus}" }
+                    }
+                  ) {
+                    projectV2Item {
+                      id
+                    }
+                  }
+                  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)